server/routes/login.js

```diff
@@ -40,11 +40,7 @@ module.exports = () => (router) => {
       const inputTwoFactorCode = utilities.createTwoFactorAuthenticationHash(req.body.code)
 
       if (inputTwoFactorCode === userAuthenticationDetails[0].TwoFactorAuthenticationHash) {
-        req.user.employeeName = await getStaffMemberEmployeeName(
-          userAuthenticationDetails[0].ApiUrl,
-          utilities.getStartMonth(),
-          req.user.token,
-        )
+        req.user.employeeName = jwtDecode(req.user.token).name
 
         await userAuthenticationService.updateUserSessionExpiryAndLastLoginDateTime(
           req.user.username,
```

Comment:
I know this is existing code, but is there a reason we are adding the value to the request object, then passing it to a function here? I may be missing something but it seems unusal to mutate the request object, unless we specifically need it for some pre response middleware. If not declaring the name as a variable would be more readable here.

Reply:
I'm not sure why we're mutating the request object but req.user.employeeName and req.user.username are different. "employeeName" is like "Nick McMahon" and "username" is my Quantum Id

Reply:
I've been reading through the codebase to see if it's used anywhere, and I can't seem to find it, but it's hard to say without any preexisting tests. Happy to take your call on whether this should be changed or moved elsewhere.